superset-frontend/src/components/DropdownButton/DropdownButton.stories.tsx

```diff
@@ -59,7 +59,8 @@ InteractiveDropdownButton.args = {
 InteractiveDropdownButton.argTypes = {
   placement: {
     defaultValue: 'top',
-    control: { type: 'select', options: PLACEMENTS },
+    control: { type: 'select' },
+    options: PLACEMENTS,
```

Comment:
Probably works great, but this thing totally crashes, as it expects to receive only a single child element.

Reply:
I don't see any changes but no crashes either, weird.